### PR TITLE
fixed loss normalization, added some hyperparams to args

### PR DESCRIPTION
**Pull request by domplexity**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -88,10 +92,10 @@ def train(input, target):
             torch.nn.utils.clip_grad_norm(model.parameters(), 0.5)
         optimizer.step()
 
-        return loss.data[0] / input.size()[0]
+        return loss.data[0] / input.size(1)  # normalize loss by sequence_length
```
Comment (pankessel) [inline train.py:95]: Wouldn't it be better to normalize by both batch_size and sequence_length?

**Review by pankessel — APPROVED** [review]
